app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(32, 6)
```

Comment:
Consider directly asserting the function call without storing the result in a variable, as per the checklist recommendation: 'Don't create unnecessary variables for simple tests.'